Assets/HoloToolkit/Input/Tests/Scripts/SphereGlobalKeywords.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.InputModule.Tests
+{
+    public class SphereGlobalKeywords : MonoBehaviour, ISpeechHandler
+    {
+        private void Start()
+        {
+        }
```

Comment:
Removing it makes the script enable checkbox disappear but we aren't handling it anyway and this is just test code...